flask_app/flask_main.py

```diff
@@ -10,100 +10,25 @@
 from pandas_code.mapping import template_mapping
 from pandas_code.parse_template import parse_template
 
+from flask_app.api.config import UPLOAD_FOLDER
+import flask_app.api.views as views
 
-ALLOWED_EXTENSIONS = {'csv'}
-
-UPLOAD_FOLDER='data/'
 
 def create_app():
 
    app = Flask(__name__, template_folder='templates')
    app.config['UPLOAD_FOLDER'] = UPLOAD_FOLDER
    generator = code_generator.CodeGenerator(template_mapping, parse_template)
```

Comment:
I think since generator was moved to api/generator.py, this line should be removed (along with the template_mapping and parse_template imports)

Reply:
Correct, I forgot this line along with a few unused imports. Will fix shortly.